### DISCUSSION TITLE
Split into Two

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7512] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

How do I split images like these into two by dividing the width by 2 while keeping the height for the output files?

![image_016](https://github.com/user-attachments/assets/b722968b-60b0-4647-837b-cb8378339871)
![10_image_009](https://github.com/user-attachments/assets/25971781-6142-40c5-a512-8dad72777443)

**Comment by snibgo** [comment]
To divide into 2 equal pieces horizontally, and 1 piece vertically:
```
magick in.png -crop 2x1@ out.png
```
This makes outputs `out-0.png` and `out-1.png`.

See https://imagemagick.org/script/command-line-options.php#crop

**Reply by Jay123210599** [reply]
How do I do this to multiple images at once?

**Comment by snibgo** [comment]
```
magick mogrify -crop 2x1@ *.png
```
